challenges/classes.js

```diff
@@ -3,9 +3,47 @@
 // Test your volume and surfaceArea methods by uncommenting the logs below:
 // console.log(cuboid.volume()); // 100
 // console.log(cuboid.surfaceArea()); // 130
-
+class cuboidMaker{
+  constructor(dimensions){
+    this.length = dimensions.length;
+    this.width = dimensions.width;
+    this.height = dimensions.height;
+  }
+  volume(length,width,height){
+    return(this.length * this.width * this.height);
+  }
+  surfaceArea(length,width,height){
+    return(this.length * this.width + this.length* this.height + this.width * this.height);
+  }
+}
+const cuboid = new cuboidMaker({
+  length : 4,
+  width : 5,
+  height : 5
+});
+//Tests
+console.log(cuboid.volume()); 
+console.log(cuboid.surfaceArea()); 
 /* Stretch Task:
 
-  Extend the base class CuboidMaker with a sub class called CubeMaker.  Find out the formulas for volume and surface area for cubes and create those methods as well.  Create a new cube object and log out the results of your new cube.
+  Extend the base class CuboidMaker with a sub class called CubeMaker.  Find out the formulas for volume and surface area for cubes and create those methods as well.  
+  Create a new cube object and log out the results of your new cube.
 
+*/
+/*
+class CubeMaker extends cuboidMaker{
+  constructor(attributes){
+    super(cuboidMaker);
+  }
+  cubeSurfaceArea(length){
```

Comment:
fyi, you could name this just surfaceArea, and have it replace the method that it's inheriting, since this is an optimized version of it.

Also, you could do the same thing with volume(), and use this.length ** 3